### PR TITLE
fix(ZNTA-54): Allow delete language

### PR DESCRIPTION
**Pull request by aeng**
https://zanata.atlassian.net/browse/ZNTA-54

### CODE REVIEW

File: zanata-war/src/main/resources/messages.properties [hunk 1]
```diff
@@ -361,7 +361,7 @@ jsf.project.ViewIn=View in {0}
 jsf.tooltip.More=More
 jsf.tooltip.ShowExample=Show example
 jsf.tooltip.HideExample=Hide example
-jsf.tooltip.TranslateOptions=Translate Options
+jsf.tooltip.Translated=Translate Options
```
Comment (carlosmunoz) [inline zanata-war/src/main/resources/messages.properties]: why this change? seems like the old name was more accurate

File: zanata-war/src/main/java/org/zanata/service/impl/LocaleServiceImpl.java [hunk 2]
```diff
@@ -35,6 +33,8 @@
 import javax.enterprise.context.RequestScoped;
 import javax.inject.Inject;
 import javax.inject.Named;
+
+import org.apache.deltaspike.jpa.api.transaction.Transactional;
```
Comment (carlosmunoz) [inline zanata-war/src/main/java/org/zanata/service/impl/LocaleServiceImpl.java]: potentially unnecessary import?

File: zanata-war/src/main/resources/messages.properties (hunk 1 above)
Comment (aeng) [inline zanata-war/src/main/resources/messages.properties]: Keypress accident. revert it back.

**Comment by carlosmunoz** [comment]
:+1: reviewed

**Comment by aeng** [comment]
@zanata-jenkins retest this please

**Comment by djansen-redhat** [comment]
I tried to remove Angika, encountered an error: http://pastebin.test.redhat.com/365983

**Comment by aeng** [comment]
> Edit: This is a result of the 'cannot remove because of a constraint' issue that also affects deleting users. In the case of "oops, I messed up creating a language" this feature works.


https://zanata.atlassian.net/browse/ZNTA-1035

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested
